Cargo.lock

```diff
@@ -2,9 +2,9 @@
 # It is not intended for manual editing.
 [[package]]
 name = "aho-corasick"
-version = "0.7.13"
+version = "0.7.15"
```

Comment:
Please drop the lockfile changes from this PR.  Or is there a specific reason you had to update dependencies?